### PR TITLE
Develop webber

### PR DESCRIPTION
**Pull request by webbertakken**
Add configurable base layer and overlays, introduced WMS.

### CODE REVIEW

File: Resources/public/js/leaflet-embed.js [hunk 1]
```diff
@@ -115,6 +117,82 @@ CuriousMap.prototype.clearFormFields = function () {
...
+ * Create and return a TileLayer
+ */
+CuriousMap.prototype.createTileLayer = function (settings) {
+  return new L.TileLayer(
+    settings.url,
+    {
+      minZoom: settings.minZoom || 1,
+      maxZoom: settings.maxZoom || 20,
+      attribution: settings.attribution || '',
+      subdomains: settings.subdomains || 'abc'
```
Comment (webbertakken) [inline Resources/public/js/leaflet-embed.js]: but isn't, it's the default from leaflet.

File: Resources/views/Form/fields.html.twig [hunk 2]
```diff
@@ -150,5 +126,30 @@
...
+      <script src="{{ asset('bundles/curiousincmapformtype/js/leaflet-embed.js') }}"></script>
+      <script>
+        var mapView = mapView || {};
+        mapView['{{ id }}'] = new CuriousMap(
+          {
+            formId: '{{ id }}',
+            defaults: {{ defaults|json_encode|raw }},
+            fields: {{ fields|json_encode|raw }},
+            baseLayer: {{ base_layer|json_encode|raw }},
+            overlays: {{ overlays|json_encode|raw }},
```
Comment (J063) [inline Resources/views/Form/fields.html.twig]: @{c34f1ffd-de5f-44c1-b8f6-9ea92d34251a,Dick Webbink | Dation}|@{c34f1ffd-de5f-44c1-b8f6-9ea92d34251a,Dick Webbink | Dation}|@{c34f1ffd-de5f-44c1-b8f6-9ea92d34251a,Dick Webbink | Dation}|@{c34f1ffd-de5f-44c1-b8f6-9ea92d34251a,Dick Webbink | Dation}

Getting an 'unneeded comma' here.

File: README.md [hunk 3]
```diff
@@ -38,14 +43,14 @@ class AppKernel extends Kernel
 }
 ```
 
-#### 3) Configure the Bundle
+#### 4) Configure the Bundle
 Add the following configuration to `app/config/config.yml` and change it to needs or wishes accordingly:
 ```yaml
 curious_inc_map_form_type:
```
Comment (J063) [inline README.md]: @{afd51144-4910-4aab-9fd3-decb868f8a51,webber} 

(1/1) InvalidConfigurationException
Unrecognized option "example" under "curious_inc_map_form_type"

I think an update is in order here.

**Comment by webbertakken** [comment]
Please review again

**Pull request merged**